db/migrate/20200917005322_create_items.rb

```diff
@@ -0,0 +1,12 @@
+class CreateItems < ActiveRecord::Migration[6.0]
+  def change
+    create_table :items do |t|
+      t.string :new
```

Comment:
Are there any other fields we might want on this table? Thinking about things like the type of media (is it a youtube video, library book, online article, etc.), the url if it's an online resource, the date it was made?